docs/questions/visualizations/table.md

```diff
@@ -36,6 +52,8 @@ To change the width of a column, you can click and drag the edges of the column'
 
 ## Column heading options for filtering and summarizing
```

Comment:
Can we also mention cell click actions like filters etc? similarly to header options - just a sentence, but i think that's pretty difficult to discover on your own